com.librumreader.librum.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: com.librumreader.librum
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: librum
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
```

Comment:
```suggestion
  - --socket=x11
```

Reply:
this assume it doesn't support wayland as you signaled by removing the wayland socket.